tests/v2/test_waku_store.nim

```diff
@@ -34,22 +27,33 @@ procSuite "Waku Store":
     discard await dialSwitch.start()
 
     var listenSwitch = newStandardSwitch(some(key))
-    listenSwitch.mount(proto)
     discard await listenSwitch.start()
 
+    let
+      proto = WakuStore.init(dialSwitch)
+      subscription = proto.subscription()
+      rpc = HistoryQuery(uuid: "1234", topics: @["topic"])
```

Comment:
Is there a TODO or follow up issue for moving request id out of the query object?

Reply:
https://github.com/status-im/nim-waku/issues/183